ExampleApp/AppDelegate.swift

```diff
@@ -236,11 +236,12 @@ class DummyMarketDataService: MarketDataService {
 }
 
 // Only implement this protocol if you need to stream your own market data
-class DummyStreamingMarketDataService: StreamingMarketDataService {
+public class DummyStreamingMarketDataService: StreamingMarketDataService {
```

Comment:
Why public? We don't want partners to start using this market data service ;) Or do you need to make it public for `ExampleApp`?

Reply:
I thought it needed to be public for `ExampleViewController` toggle button 😅. Reverting it to default since it's fine without it.